src/components/Form/Form.md

```diff
@@ -131,6 +131,14 @@ function validateIllegal(x) {
   return x === 'illegal' ? 'That\'s an illegal option! Choose another.' : ''
 }
 
+function validateExists(x) {
```

Comment:
I suspect something like this should actually replace validateTruthy and we may need to do similar stuff in other fields too (not just button select groups). Maybe radio buttons, and checkboxes especially.